scripts/radar.js

```diff
@@ -1,11 +0,0 @@
-// Description
```

Comment:
While the functionality for this is technically duplicated in weather.js, it's hella broken, so can we keep this one?

Reply:
Done